src/controller.coffee

```diff
@@ -645,6 +657,9 @@ define ['droplet-helper',
   # Simple function for going through a linked-list block
   # and seeing what the innermost child is that we hit.
   Editor::hitTest = (point, block) ->
+    if @readOnly
+      return null
+
```

Comment:
For this and trackerPointIsInElement, it's not clear to me in isolation why these functions should exiting early in readOnly mode. I'm not entirely sure what a trackerPoint is, but I would expect that it is/isn't in an element independent of whether or not we're in read only mode. For hit test, my confusion might come from the fact that it's not entirely clear to me what it means to find a hit.